### PR TITLE
[Channel] Add not blank validation for channel default currency

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | https://github.com/Sylius/Sylius/pull/5565
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Page/Admin/Channel/CreatePage.php [hunk 1]
```diff
@@ -86,6 +86,14 @@ public function chooseDefaultTaxZone($taxZone)
     /**
      * {@inheritdoc}
      */
+    public function chooseDefaultLocale($locale)
+    {
+//        $this->getElement('default_locale')->selectOption($locale);
```
Comment (lchrusciel) [inline src/Sylius/Behat/Page/Admin/Channel/CreatePage.php]: Why did you leave it?

File: features/channel/managing_channels/channel_validation.feature [hunk 2]
```diff
@@ -13,7 +13,7 @@ Feature: Channel validation
         When I name it "Mobile channel"
         But I do not specify its code
         And I try to add it
-        Then I should be notified that code is required
+        Then I should be notified that "code" is required
```
Comment (pamil) [inline features/channel/managing_channels/channel_validation.feature]: Why? :-1:

File: features/channel/managing_channels/adding_channel.feature [hunk 3]
```diff
@@ -6,12 +6,14 @@ Feature: Adding a new channel
 
     Background:
         Given I am logged in as an administrator
+        And the store has currency "Euro"
```
Comment (michalmarcinkowski) [inline features/channel/managing_channels/adding_channel.feature]:
```
Given the store has currency "Euro"
And I am logged in as an administrator
```

At the beginning we should configure shop and login after everything is setup (we follow this convention everywhere).

File: features/channel/managing_channels/channel_validation.feature [hunk 4]
```diff
@@ -26,10 +26,20 @@ Feature: Channel validation
...
+
+    @ui
     Scenario: Trying to remove name from existing channel
         Given the store operates on a channel named "Web Channel"
         And I want to modify this channel
         When I remove its name
         And I try to save my changes
         Then I should be notified that name is required
         And this channel should still be named "Web Channel"
+
```
Comment (michalmarcinkowski) [inline features/channel/managing_channels/channel_validation.feature]: Redundant

**Comment by pamil** [comment]
:+1:, the same will need to be done for channel's default locale

**Comment by michalmarcinkowski** [comment]
Good job! 👍

**Pull request merged**